src/integration/blockchain/ain/services/crypto.service.ts

```diff
@@ -58,6 +64,7 @@ export class CryptoService {
   }
 
   private verifyBitcoinBased(message: string, address: string, signature: string, prefix: string | null): boolean {
+    if (this.isLightningAddress(address)) return this.verifyLightningBased(message, address, signature, prefix);
```

Comment:
Why this?

Reply:
wups missed to remove. first i have integrated it that Lightning is part of Bitcoin blockchain, after discussions we decided to create an own blockchain for it